man/mtf-env-clean.1

```diff
@@ -0,0 +1,40 @@
+.\" Copyright Petr Hracek, 2017
+.\"
+.\" This page is distributed under GPL.
+.\"
+.TH MTF-ENV-CLEAN 1 2017-11-01 "" "Linux User's Manual"
+.SH NAME
+mtf-env-clean \- cleans environment for testing containers.
+
+.SH SYNOPSIS
+\fIMODULE=docker\/\fR
+.B mtf-env-clean
+
+\fIMODULE=rpm\/\fR
+.B mtf-env-clean
+
+\fIMODULE=nspawn\/\fR
+.B mtf-env-clean
+
+.SH DESCRIPTION
+.PP
+\fBmtf-env-clean\fP is the binary file for cleaning environment of Meta-Test-Family.
+
+.PP
+\fIMODULE=docker\/\fR stops docker service.
+
+.PP
+\fIMODULE=rpm\/\fR does not do cleanup as it can destroy this machine.
```

Comment:
Rather `does not do any cleanup, as that could potentially uninstall essential packages and therefore damage this machine.`